packages/select/test/select.test.js

```diff
@@ -402,29 +384,6 @@ describe('vaadin-select', () => {
         expect(overlay.opened).to.be.false;
       });
 
-      it('should restore focused state on closing the overlay if phone', async () => {
-        select._phone = true;
```

Comment:
The closing logic specific to `_phone` has been removed in https://github.com/vaadin/web-components/pull/3120 as part of an a11y fix, so there is no need for separate tests covering this mode anymore.